### PR TITLE
Add email notification to sqs queue

### PR DESCRIPTION
**Pull request by mannickutd**

### CODE REVIEW

File: app/notifications/rest.py [hunk 1]
```diff
@@ -7,10 +7,11 @@
...
-from app import notify_alpha_client
-from app import api_user
+from app import (notify_alpha_client, api_user)
+from app.aws_sqs import add_notification_to_queue
 from app.dao import (templates_dao, services_dao)
+from app.schemas import (
+    email_notification_schema, sms_admin_notification_schema, sms_template_notification_schema)
 import re
 
 mobile_regex = re.compile("^\\+44[\\d]{10}$")
```
Comment (servingUpAces) [inline app/notifications/rest.py]: This is no longer used.

Comment (servingUpAces) [inline app/notifications/rest.py]: There are some unused imports

File: app/schemas.py [hunk 2]
```diff
@@ -1,7 +1,11 @@
+import re
 from flask_marshmallow.fields import fields
 from . import ma
 from . import models
-from marshmallow import post_load, ValidationError
+from marshmallow_sqlalchemy.fields import Related
```
Comment (servingUpAces) [inline app/schemas.py]: unused import

**Comment by mannickutd** [comment]
Should be good now.

**Pull request merged**